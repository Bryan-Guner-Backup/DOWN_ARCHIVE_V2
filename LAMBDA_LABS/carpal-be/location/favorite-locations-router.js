const express = require("express");
const router = express.Router();
const { verifyToken, validateUserToken } = require("../Middleware/auth");
const { FavoriteLocations } = require("../Classes/favorite-locations");
const { checkBody, locationCheck } = require("../Middleware/locations");
const { Models } = require("../Classes/models");

//new favorite locations DB class
const FaveLocations = new FavoriteLocations();

router.get("/", async (req, res, next) => {
    try {
        res.json(await FaveLocations.getFavorites(req.user.id));
    } catch (err) {
        next(err);
    }
});

router.get("/:id", async (req, res, next) => {
    try {
        res.json(await FaveLocations.getSpecificFavorite(req.params.id));
    } catch (err) {
        next(err);
    }
});

//middleware check for address in locations DB
router.post("/", checkBody(), locationCheck(), async (req, res, next) => {
    try {
        if (req.addedLocation) {
            res.status(201).json(
                await FaveLocations.add(
                    req.user.id,
                    req.addedLocation.id,
                    req.body.name
                )
            );
        } else {
            res.status(201).json(
                await FaveLocations.add(
                    req.user.id,
                    req.location.id,
                    req.body.name
                )
            );
        }
    } catch (err) {
        next(err);
    }
});

router.delete("/:id", async (req, res, next) => {
    try {
        res.json(await FaveLocations.delete(req.params.id));
    } catch (err) {
        next(err);
    }
});

router.put("/:id", checkBody(), locationCheck(), async (req, res, next) => {
    try {
        // if location had to be added to DB, update favorite location with updated location ID
        if (req.addedLocation) {
            res.json(
                await FaveLocations.update(req.params.id, {
                    location_id: req.addedLocation.id,
                    name: req.body.name
                })
            );
        } else {
            res.json(
                await FaveLocations.update(req.params.id, {
                    location_id: req.location.id,
                    name: req.body.name
                })
            );
        }
    } catch (err) {
        next(err);
    }
});

module.exports = router;
