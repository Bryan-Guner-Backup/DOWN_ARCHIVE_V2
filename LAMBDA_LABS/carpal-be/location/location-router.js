const { Models } = require("../Classes/models");
const express = require("express");
const favoritesRouter = require("./favorite-locations-router");
const router = express.Router();
const { checkBody } = require("../Middleware/locations");

//new locations DB class
const locations = new Models("locations");

router.use("/favorites", favoritesRouter);

router.get("/", async (req, res, next) => {
    try {
        res.json(await locations.findAll());
    } catch (err) {
        next(err);
    }
});

//needs middleware validation for valid ID
router.get("/:id", async (req, res, next) => {
    try {
        res.json(await locations.findBy({ id: req.params.id }));
    } catch (err) {
        next(err);
    }
});

//needs middleware validation for req.body
router.post("/", checkBody(), async (req, res, next) => {
    try {
        res.status(201).json(await locations.add(req.body));
    } catch (err) {
        next(err);
    }
});

router.put("/:id", checkBody(), async (req, res, next) => {
    try {
        res.json(await locations.update(req.params.id, req.body));
    } catch (err) {
        next(err);
    }
});

//needs middleware validation for valid ID
router.delete("/:id", async (req, res, next) => {
    try {
        res.json(await locations.delete(req.params.id));
    } catch (err) {
        next(err);
    }
});

module.exports = router;
