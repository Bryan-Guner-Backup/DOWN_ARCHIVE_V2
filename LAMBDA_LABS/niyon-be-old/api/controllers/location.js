const models = require('../../database/models');
const response = require('../helpers/response');

module.exports = {
  async getCountries(req, res, next) {
    try {
      const countries = await models.Locations.findAll({
        attributes: ['country_name'],
        returning: true
      });
      return response.success(res, 200, countries);
    } catch (error) {
      return next({ message: error.message });
    }
  },
  // currently not in use
  async getCitiesByCountries(req, res, next) {
    try {
      const { country } = req.params;
      const cities = await models.Locations.findAll({
        attributes: ['city_name'],
        where: { country_name: country },
        returning: true
      });
      if (!cities.length) {
        return response.success(res, 200, 'No cities saved in the country');
      }
      return response.success(res, 200, cities);
    } catch (error) {
      return next({ message: error.message });
    }
  },

  async findOrCreateLocation(req, res, next) {
    try {
      const { cityName, countryName } = req.body;
      const locations = await models.Locations.findOrCreate({
        where: { country_name: countryName, city_name: cityName },
        attributes: ['id']
      });
      const locationId = locations[0].dataValues.id;
      return response.success(res, 201, locationId);
    } catch (error) {
      return next({ message: error.message });
    }
  }
};
