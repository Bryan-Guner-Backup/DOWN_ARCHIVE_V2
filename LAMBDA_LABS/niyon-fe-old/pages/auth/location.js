import Location from '../../components/auth/Location';

const location = () => (
  <div>
    <Location />
  </div>
);

export default location;
