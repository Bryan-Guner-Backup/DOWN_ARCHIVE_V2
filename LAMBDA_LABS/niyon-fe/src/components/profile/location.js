export const location = [
  {
    value: 1,
    label: 'Unknown'
  },
  {
    value: 2,
    label: 'Lagos, Nigeria'
  },
  {
    value: 3,
    label: 'Kano, Nigeria'
  },
  {
    value: 4,
    label: 'Ibadan, Nigeria'
  },
  {
    value: 5,
    label: 'Kaduna, Nigeria'
  },
  {
    value: 6,
    label: 'Port Harcourt, Nigeria'
  },
  {
    value: 7,
    label: 'Benin, Nigeria'
  },
  {
    value: 8,
    label: 'Maiduguri, Nigeria'
  },
  {
    value: 9,
    label: 'Zaria, Nigeria'
  },
  {
    value: 10,
    label: 'Aba, Nigeria'
  },
  {
    value: 11,
    label: 'Ilorin, Nigeria'
  },
  {
    value: 12,
    label: 'Jos, Nigeria'
  },
  {
    value: 13,
    label: 'Ogbomosho, Nigeria'
  },
  {
    value: 14,
    label: 'Ovo, Nigeria'
  },
  {
    value: 15,
    label: 'Enugu, Nigeria'
  },
  {
    value: 16,
    label: 'Abeokuta, Nigeria'
  },
  {
    value: 17,
    label: 'Onitsha, Nigeria'
  },
  {
    value: 18,
    label: 'Warri, Nigeria'
  },
  {
    value: 19,
    label: 'Sokoto, Nigeria'
  },
  {
    value: 20,
    label: 'Okene, Nigeria'
  },
  {
    value: 21,
    label: 'Calabar, Nigeria'
  },
  {
    value: 22,
    label: 'Oshogbo, Nigeria'
  },
  {
    value: 23,
    label: 'Katsina, Nigeria'
  },
  {
    value: 24,
    label: 'Akure, Nigeria'
  },
  {
    value: 25,
    label: 'Bobo Dioulasso, Nigeria'
  },
  {
    value: 26,
    label: 'Ife, Nigeria'
  },
  {
    value: 27,
    label: 'Bauchi, Nigeria'
  },
  {
    value: 28,
    label: 'Iseyin, Nigeria'
  },
  {
    value: 29,
    label: 'Minna, Nigeria'
  },
  {
    value: 30,
    label: 'Makurdi, Nigeria'
  },
  {
    value: 31,
    label: 'Owo, Nigeria'
  },
  {
    value: 32,
    label: 'Ado, Nigeria'
  },
  {
    value: 33,
    label: 'Ilesha, Nigeria'
  },
  {
    value: 34,
    label: 'Gombe, Nigeria'
  },
  {
    value: 35,
    label: 'Umuahia, Nigeria'
  },
  {
    value: 36,
    label: 'Ondo, Nigeria'
  },
  {
    value: 37,
    label: 'Damaturu, Nigeria'
  },
  {
    value: 38,
    label: 'Jimeta, Nigeria'
  },
  {
    value: 39,
    label: 'Ikot Ekpene, Nigeria'
  },
  {
    value: 40,
    label: 'Gusau, Nigeria'
  },
  {
    value: 41,
    label: 'Mubi, Nigeria'
  },
  {
    value: 42,
    label: 'Shagumu, Nigeria'
  },
  {
    value: 43,
    label: 'Owerri, Nigeria'
  },
  {
    value: 44,
    label: 'Ugep, Nigeria'
  },
  {
    value: 45,
    label: 'Ijebu Ode, Nigeria'
  },
  {
    value: 46,
    label: 'Ise, Nigeria'
  },
  {
    value: 47,
    label: 'Gboko, Nigeria'
  },
  {
    value: 48,
    label: 'Ilawe, Nigeria'
  },
  {
    value: 49,
    label: 'Ikare, Nigeria'
  },
  {
    value: 50,
    label: 'Abuja, Nigeria'
  },
  {
    value: 51,
    label: 'Okpoko, Nigeria'
  },
  {
    value: 52,
    label: 'Awka, Nigeria'
  },
  {
    value: 53,
    label: 'Sapele, Nigeria'
  },
  {
    value: 54,
    label: 'Ila, Nigeria'
  },
  {
    value: 55,
    label: 'Shaki, Nigeria'
  },
  {
    value: 56,
    label: 'Ijero, Nigeria'
  },
  {
    value: 57,
    label: 'Otukpo, Nigeria'
  },
  {
    value: 58,
    label: 'Kishi, Nigeria'
  },
  {
    value: 59,
    label: 'Bugama, Nigeria'
  },
  {
    value: 60,
    label: 'Funtua, Nigeria'
  },
  {
    value: 61,
    label: 'Abakaliki, Nigeria'
  },
  {
    value: 62,
    label: 'Gbongan, Nigeria'
  },
  {
    value: 63,
    label: 'Lafia, Nigeria'
  },
  {
    value: 64,
    label: 'Igboho, Nigeria'
  },
  {
    value: 65,
    label: 'Amaigbo, Nigeria'
  },
  {
    value: 66,
    label: 'Gashua, Nigeria'
  },
  {
    value: 67,
    label: 'Offa, Nigeria'
  },
  {
    value: 68,
    label: 'Jalingo, Nigeria'
  },
  {
    value: 69,
    label: 'Bama, Nigeria'
  },
  {
    value: 70,
    label: 'Uvo, Nigeria'
  },
  {
    value: 71,
    label: 'Uromi, Nigeria'
  },
  {
    value: 72,
    label: 'Nsukka, Nigeria'
  },
  {
    value: 73,
    label: 'Okigwe, Nigeria'
  },
  {
    value: 74,
    label: 'Modakeke, Nigeria'
  },
  {
    value: 75,
    label: 'Abidjan, Ivory Coast'
  },
  {
    value: 76,
    label: 'Bouake, Ivory Coast'
  },
  {
    value: 77,
    label: 'Daloa, Ivory Coast'
  },
  {
    value: 78,
    label: 'Yamoussoukro, Ivory Coast'
  },
  {
    value: 79,
    label: 'Korhogo, Ivory Coast'
  },
  {
    value: 80,
    label: 'San Pedro, Ivory Coast'
  },
  {
    value: 81,
    label: 'Divo, Ivory Coast'
  },
  {
    value: 82,
    label: 'Man, Ivory Coast'
  },
  {
    value: 83,
    label: 'Gagnoa, Ivory Coast'
  },
  {
    value: 84,
    label: 'Dakar, Senegal'
  },
  {
    value: 85,
    label: 'Thies, Senegal'
  },
  {
    value: 86,
    label: 'Kaolack, Senegal'
  },
  {
    value: 87,
    label: 'Ziguinchor, Senegal'
  },
  {
    value: 88,
    label: 'Saint Louis, Senegal'
  },
  {
    value: 89,
    label: 'Mbour, Senegal'
  },
  {
    value: 90,
    label: 'Diourbel, Senegal'
  },
  {
    value: 91,
    label: 'Accra, Ghana'
  },
  {
    value: 92,
    label: 'Kumasi, Ghana'
  },
  {
    value: 93,
    label: 'Tamale, Ghana'
  },
  {
    value: 94,
    label: 'Obuasi, Ghana'
  },
  {
    value: 95,
    label: 'Conakry, Guinea'
  },
  {
    value: 96,
    label: 'Kankan, Guinea'
  },
  {
    value: 97,
    label: 'Bissua, Guinea-Bissau'
  },
  {
    value: 98,
    label: 'Douala, Cameroon'
  },
  {
    value: 99,
    label: 'Yaounde, Cameroon'
  },
  {
    value: 100,
    label: 'Garoua, Cameroon'
  },
  {
    value: 101,
    label: 'Maroua, Cameroon'
  },
  {
    value: 102,
    label: 'Bafoussan, Cameroon'
  },
  {
    value: 103,
    label: 'Ngaoundere, Cameroon'
  },
  {
    value: 104,
    label: 'Bamenda, Cameroon'
  },
  {
    value: 105,
    label: 'Kaele, Cameroon'
  },
  {
    value: 106,
    label: 'Kumba, Cameroon'
  },
  {
    value: 107,
    label: 'Freetown, Sierra Leone'
  },
  {
    value: 108,
    label: 'Koidu, Sierra Leone'
  },
  {
    value: 109,
    label: 'Makeni, Sierra Leone'
  },
  {
    value: 110,
    label: 'Ouagadougou, Burkina Faso'
  },
  {
    value: 111,
    label: 'Bobo Diolasso, Burkina Faso'
  },
  {
    value: 112,
    label: 'Cotonou, Benin'
  },
  {
    value: 113,
    label: 'Porto Novo, Benin'
  },
  {
    value: 114,
    label: 'Parakou, Benin'
  },
  {
    value: 115,
    label: 'Djougou, Benin'
  },
  {
    value: 116,
    label: 'Bohicon, Benin'
  },
  {
    value: 117,
    label: 'Kandi, Benin'
  },
  {
    value: 118,
    label: 'Abomey, Benin'
  },
  {
    value: 119,
    label: 'Natitingou, Benin'
  },
  {
    value: 120,
    label: 'Lome, Togo'
  },
  {
    value: 121,
    label: 'Sokode, Togo'
  },
  {
    value: 122,
    label: 'Monrovie, Liberia'
  },
  {
    value: 123,
    label: 'Ajdabivah, Liberia'
  },
  {
    value: 124,
    label: 'Libreville, Gabon'
  },
  {
    value: 125,
    label: 'Port Gentil. Gabon'
  },
  {
    value: 126,
    label: 'Serekunda, Gambia'
  },
  {
    value: 127,
    label: 'Unknown'
  },
  {
    value: 128,
    label: 'Lagos, Nigeria'
  },
  {
    value: 129,
    label: 'Kano, Nigeria'
  },
  {
    value: 130,
    label: 'Ibadan, Nigeria'
  },
  {
    value: 131,
    label: 'Kaduna, Nigeria'
  },
  {
    value: 132,
    label: 'Port Harcourt, Nigeria'
  },
  {
    value: 133,
    label: 'Benin, Nigeria'
  },
  {
    value: 134,
    label: 'Maiduguri, Nigeria'
  },
  {
    value: 135,
    label: 'Zaria, Nigeria'
  },
  {
    value: 136,
    label: 'Aba, Nigeria'
  },
  {
    value: 137,
    label: 'Ilorin, Nigeria'
  },
  {
    value: 138,
    label: 'Jos, Nigeria'
  },
  {
    value: 139,
    label: 'Ogbomosho, Nigeria'
  },
  {
    value: 140,
    label: 'Ovo, Nigeria'
  },
  {
    value: 141,
    label: 'Enugu, Nigeria'
  },
  {
    value: 142,
    label: 'Abeokuta, Nigeria'
  },
  {
    value: 143,
    label: 'Onitsha, Nigeria'
  },
  {
    value: 144,
    label: 'Warri, Nigeria'
  },
  {
    value: 145,
    label: 'Sokoto, Nigeria'
  },
  {
    value: 146,
    label: 'Okene, Nigeria'
  },
  {
    value: 147,
    label: 'Calabar, Nigeria'
  },
  {
    value: 148,
    label: 'Oshogbo, Nigeria'
  },
  {
    value: 149,
    label: 'Katsina, Nigeria'
  },
  {
    value: 150,
    label: 'Akure, Nigeria'
  },
  {
    value: 151,
    label: 'Bobo Dioulasso, Nigeria'
  },
  {
    value: 152,
    label: 'Ife, Nigeria'
  },
  {
    value: 153,
    label: 'Bauchi, Nigeria'
  },
  {
    value: 154,
    label: 'Iseyin, Nigeria'
  },
  {
    value: 155,
    label: 'Minna, Nigeria'
  },
  {
    value: 156,
    label: 'Makurdi, Nigeria'
  },
  {
    value: 157,
    label: 'Owo, Nigeria'
  },
  {
    value: 158,
    label: 'Ado, Nigeria'
  },
  {
    value: 159,
    label: 'Ilesha, Nigeria'
  },
  {
    value: 160,
    label: 'Gombe, Nigeria'
  },
  {
    value: 161,
    label: 'Umuahia, Nigeria'
  },
  {
    value: 162,
    label: 'Ondo, Nigeria'
  },
  {
    value: 163,
    label: 'Damaturu, Nigeria'
  },
  {
    value: 164,
    label: 'Jimeta, Nigeria'
  },
  {
    value: 165,
    label: 'Ikot Ekpene, Nigeria'
  },
  {
    value: 166,
    label: 'Gusau, Nigeria'
  },
  {
    value: 167,
    label: 'Mubi, Nigeria'
  },
  {
    value: 168,
    label: 'Shagumu, Nigeria'
  },
  {
    value: 169,
    label: 'Owerri, Nigeria'
  },
  {
    value: 170,
    label: 'Ugep, Nigeria'
  },
  {
    value: 171,
    label: 'Ijebu Ode, Nigeria'
  },
  {
    value: 172,
    label: 'Ise, Nigeria'
  },
  {
    value: 173,
    label: 'Gboko, Nigeria'
  },
  {
    value: 174,
    label: 'Ilawe, Nigeria'
  },
  {
    value: 175,
    label: 'Ikare, Nigeria'
  },
  {
    value: 176,
    label: 'Abuja, Nigeria'
  },
  {
    value: 177,
    label: 'Okpoko, Nigeria'
  },
  {
    value: 178,
    label: 'Awka, Nigeria'
  },
  {
    value: 179,
    label: 'Sapele, Nigeria'
  },
  {
    value: 180,
    label: 'Ila, Nigeria'
  },
  {
    value: 181,
    label: 'Shaki, Nigeria'
  },
  {
    value: 182,
    label: 'Ijero, Nigeria'
  },
  {
    value: 183,
    label: 'Otukpo, Nigeria'
  },
  {
    value: 184,
    label: 'Kishi, Nigeria'
  },
  {
    value: 185,
    label: 'Bugama, Nigeria'
  },
  {
    value: 186,
    label: 'Funtua, Nigeria'
  },
  {
    value: 187,
    label: 'Abakaliki, Nigeria'
  },
  {
    value: 188,
    label: 'Gbongan, Nigeria'
  },
  {
    value: 189,
    label: 'Lafia, Nigeria'
  },
  {
    value: 190,
    label: 'Igboho, Nigeria'
  },
  {
    value: 191,
    label: 'Amaigbo, Nigeria'
  },
  {
    value: 192,
    label: 'Gashua, Nigeria'
  },
  {
    value: 193,
    label: 'Offa, Nigeria'
  },
  {
    value: 194,
    label: 'Jalingo, Nigeria'
  },
  {
    value: 195,
    label: 'Bama, Nigeria'
  },
  {
    value: 196,
    label: 'Uvo, Nigeria'
  },
  {
    value: 197,
    label: 'Uromi, Nigeria'
  },
  {
    value: 198,
    label: 'Nsukka, Nigeria'
  },
  {
    value: 199,
    label: 'Okigwe, Nigeria'
  },
  {
    value: 200,
    label: 'Modakeke, Nigeria'
  },
  {
    value: 201,
    label: 'Abidjan, Ivory Coast'
  },
  {
    value: 202,
    label: 'Bouake, Ivory Coast'
  },
  {
    value: 203,
    label: 'Daloa, Ivory Coast'
  },
  {
    value: 204,
    label: 'Yamoussoukro, Ivory Coast'
  },
  {
    value: 205,
    label: 'Korhogo, Ivory Coast'
  },
  {
    value: 206,
    label: 'San Pedro, Ivory Coast'
  },
  {
    value: 207,
    label: 'Divo, Ivory Coast'
  },
  {
    value: 208,
    label: 'Man, Ivory Coast'
  },
  {
    value: 209,
    label: 'Gagnoa, Ivory Coast'
  },
  {
    value: 210,
    label: 'Dakar, Senegal'
  },
  {
    value: 211,
    label: 'Thies, Senegal'
  },
  {
    value: 212,
    label: 'Kaolack, Senegal'
  },
  {
    value: 213,
    label: 'Ziguinchor, Senegal'
  },
  {
    value: 214,
    label: 'Saint Louis, Senegal'
  },
  {
    value: 215,
    label: 'Mbour, Senegal'
  },
  {
    value: 216,
    label: 'Diourbel, Senegal'
  },
  {
    value: 217,
    label: 'Accra, Ghana'
  },
  {
    value: 218,
    label: 'Kumasi, Ghana'
  },
  {
    value: 219,
    label: 'Tamale, Ghana'
  },
  {
    value: 220,
    label: 'Obuasi, Ghana'
  },
  {
    value: 221,
    label: 'Conakry, Guinea'
  },
  {
    value: 222,
    label: 'Kankan, Guinea'
  },
  {
    value: 223,
    label: 'Bissua, Guinea-Bissau'
  },
  {
    value: 224,
    label: 'Douala, Cameroon'
  },
  {
    value: 225,
    label: 'Yaounde, Cameroon'
  },
  {
    value: 226,
    label: 'Garoua, Cameroon'
  },
  {
    value: 227,
    label: 'Maroua, Cameroon'
  },
  {
    value: 228,
    label: 'Bafoussan, Cameroon'
  },
  {
    value: 229,
    label: 'Ngaoundere, Cameroon'
  },
  {
    value: 230,
    label: 'Bamenda, Cameroon'
  },
  {
    value: 231,
    label: 'Kaele, Cameroon'
  },
  {
    value: 232,
    label: 'Kumba, Cameroon'
  },
  {
    value: 233,
    label: 'Freetown, Sierra Leone'
  },
  {
    value: 234,
    label: 'Koidu, Sierra Leone'
  },
  {
    value: 235,
    label: 'Makeni, Sierra Leone'
  },
  {
    value: 236,
    label: 'Ouagadougou, Burkina Faso'
  },
  {
    value: 237,
    label: 'Bobo Diolasso, Burkina Faso'
  },
  {
    value: 238,
    label: 'Cotonou, Benin'
  },
  {
    value: 239,
    label: 'Porto Novo, Benin'
  },
  {
    value: 240,
    label: 'Parakou, Benin'
  },
  {
    value: 241,
    label: 'Djougou, Benin'
  },
  {
    value: 242,
    label: 'Bohicon, Benin'
  },
  {
    value: 243,
    label: 'Kandi, Benin'
  },
  {
    value: 244,
    label: 'Abomey, Benin'
  },
  {
    value: 245,
    label: 'Natitingou, Benin'
  },
  {
    value: 246,
    label: 'Lome, Togo'
  },
  {
    value: 247,
    label: 'Sokode, Togo'
  },
  {
    value: 248,
    label: 'Monrovie, Liberia'
  },
  {
    value: 249,
    label: 'Ajdabivah, Liberia'
  },
  {
    value: 250,
    label: 'Libreville, Gabon'
  },
  {
    value: 251,
    label: 'Port Gentil. Gabon'
  },
  {
    value: 252,
    label: 'Serekunda, Gambia'
  },
  {
    value: 253,
    label: 'Unknown'
  },
  {
    value: 254,
    label: 'Lagos, Nigeria'
  },
  {
    value: 255,
    label: 'Kano, Nigeria'
  },
  {
    value: 256,
    label: 'Ibadan, Nigeria'
  },
  {
    value: 257,
    label: 'Kaduna, Nigeria'
  },
  {
    value: 258,
    label: 'Port Harcourt, Nigeria'
  },
  {
    value: 259,
    label: 'Benin, Nigeria'
  },
  {
    value: 260,
    label: 'Maiduguri, Nigeria'
  },
  {
    value: 261,
    label: 'Zaria, Nigeria'
  },
  {
    value: 262,
    label: 'Aba, Nigeria'
  },
  {
    value: 263,
    label: 'Ilorin, Nigeria'
  },
  {
    value: 264,
    label: 'Jos, Nigeria'
  },
  {
    value: 265,
    label: 'Ogbomosho, Nigeria'
  },
  {
    value: 266,
    label: 'Ovo, Nigeria'
  },
  {
    value: 267,
    label: 'Enugu, Nigeria'
  },
  {
    value: 268,
    label: 'Abeokuta, Nigeria'
  },
  {
    value: 269,
    label: 'Onitsha, Nigeria'
  },
  {
    value: 270,
    label: 'Warri, Nigeria'
  },
  {
    value: 271,
    label: 'Sokoto, Nigeria'
  },
  {
    value: 272,
    label: 'Okene, Nigeria'
  },
  {
    value: 273,
    label: 'Calabar, Nigeria'
  },
  {
    value: 274,
    label: 'Oshogbo, Nigeria'
  },
  {
    value: 275,
    label: 'Katsina, Nigeria'
  },
  {
    value: 276,
    label: 'Akure, Nigeria'
  },
  {
    value: 277,
    label: 'Bobo Dioulasso, Nigeria'
  },
  {
    value: 278,
    label: 'Ife, Nigeria'
  },
  {
    value: 279,
    label: 'Bauchi, Nigeria'
  },
  {
    value: 280,
    label: 'Iseyin, Nigeria'
  },
  {
    value: 281,
    label: 'Minna, Nigeria'
  },
  {
    value: 282,
    label: 'Makurdi, Nigeria'
  },
  {
    value: 283,
    label: 'Owo, Nigeria'
  },
  {
    value: 284,
    label: 'Ado, Nigeria'
  },
  {
    value: 285,
    label: 'Ilesha, Nigeria'
  },
  {
    value: 286,
    label: 'Gombe, Nigeria'
  },
  {
    value: 287,
    label: 'Umuahia, Nigeria'
  },
  {
    value: 288,
    label: 'Ondo, Nigeria'
  },
  {
    value: 289,
    label: 'Damaturu, Nigeria'
  },
  {
    value: 290,
    label: 'Jimeta, Nigeria'
  },
  {
    value: 291,
    label: 'Ikot Ekpene, Nigeria'
  },
  {
    value: 292,
    label: 'Gusau, Nigeria'
  },
  {
    value: 293,
    label: 'Mubi, Nigeria'
  },
  {
    value: 294,
    label: 'Shagumu, Nigeria'
  },
  {
    value: 295,
    label: 'Owerri, Nigeria'
  },
  {
    value: 296,
    label: 'Ugep, Nigeria'
  },
  {
    value: 297,
    label: 'Ijebu Ode, Nigeria'
  },
  {
    value: 298,
    label: 'Ise, Nigeria'
  },
  {
    value: 299,
    label: 'Gboko, Nigeria'
  },
  {
    value: 300,
    label: 'Ilawe, Nigeria'
  },
  {
    value: 301,
    label: 'Ikare, Nigeria'
  },
  {
    value: 302,
    label: 'Abuja, Nigeria'
  },
  {
    value: 303,
    label: 'Okpoko, Nigeria'
  },
  {
    value: 304,
    label: 'Awka, Nigeria'
  },
  {
    value: 305,
    label: 'Sapele, Nigeria'
  },
  {
    value: 306,
    label: 'Ila, Nigeria'
  },
  {
    value: 307,
    label: 'Shaki, Nigeria'
  },
  {
    value: 308,
    label: 'Ijero, Nigeria'
  },
  {
    value: 309,
    label: 'Otukpo, Nigeria'
  },
  {
    value: 310,
    label: 'Kishi, Nigeria'
  },
  {
    value: 311,
    label: 'Bugama, Nigeria'
  },
  {
    value: 312,
    label: 'Funtua, Nigeria'
  },
  {
    value: 313,
    label: 'Abakaliki, Nigeria'
  },
  {
    value: 314,
    label: 'Gbongan, Nigeria'
  },
  {
    value: 315,
    label: 'Lafia, Nigeria'
  },
  {
    value: 316,
    label: 'Igboho, Nigeria'
  },
  {
    value: 317,
    label: 'Amaigbo, Nigeria'
  },
  {
    value: 318,
    label: 'Gashua, Nigeria'
  },
  {
    value: 319,
    label: 'Offa, Nigeria'
  },
  {
    value: 320,
    label: 'Jalingo, Nigeria'
  },
  {
    value: 321,
    label: 'Bama, Nigeria'
  },
  {
    value: 322,
    label: 'Uvo, Nigeria'
  },
  {
    value: 323,
    label: 'Uromi, Nigeria'
  },
  {
    value: 324,
    label: 'Nsukka, Nigeria'
  },
  {
    value: 325,
    label: 'Okigwe, Nigeria'
  },
  {
    value: 326,
    label: 'Modakeke, Nigeria'
  },
  {
    value: 327,
    label: 'Abidjan, Ivory Coast'
  },
  {
    value: 328,
    label: 'Bouake, Ivory Coast'
  },
  {
    value: 329,
    label: 'Daloa, Ivory Coast'
  },
  {
    value: 330,
    label: 'Yamoussoukro, Ivory Coast'
  },
  {
    value: 331,
    label: 'Korhogo, Ivory Coast'
  },
  {
    value: 332,
    label: 'San Pedro, Ivory Coast'
  },
  {
    value: 333,
    label: 'Divo, Ivory Coast'
  },
  {
    value: 334,
    label: 'Man, Ivory Coast'
  },
  {
    value: 335,
    label: 'Gagnoa, Ivory Coast'
  },
  {
    value: 336,
    label: 'Dakar, Senegal'
  },
  {
    value: 337,
    label: 'Thies, Senegal'
  },
  {
    value: 338,
    label: 'Kaolack, Senegal'
  },
  {
    value: 339,
    label: 'Ziguinchor, Senegal'
  },
  {
    value: 340,
    label: 'Saint Louis, Senegal'
  },
  {
    value: 341,
    label: 'Mbour, Senegal'
  },
  {
    value: 342,
    label: 'Diourbel, Senegal'
  },
  {
    value: 343,
    label: 'Accra, Ghana'
  },
  {
    value: 344,
    label: 'Kumasi, Ghana'
  },
  {
    value: 345,
    label: 'Tamale, Ghana'
  },
  {
    value: 346,
    label: 'Obuasi, Ghana'
  },
  {
    value: 347,
    label: 'Conakry, Guinea'
  },
  {
    value: 348,
    label: 'Kankan, Guinea'
  },
  {
    value: 349,
    label: 'Bissua, Guinea-Bissau'
  },
  {
    value: 350,
    label: 'Douala, Cameroon'
  },
  {
    value: 351,
    label: 'Yaounde, Cameroon'
  },
  {
    value: 352,
    label: 'Garoua, Cameroon'
  },
  {
    value: 353,
    label: 'Maroua, Cameroon'
  },
  {
    value: 354,
    label: 'Bafoussan, Cameroon'
  },
  {
    value: 355,
    label: 'Ngaoundere, Cameroon'
  },
  {
    value: 356,
    label: 'Bamenda, Cameroon'
  },
  {
    value: 357,
    label: 'Kaele, Cameroon'
  },
  {
    value: 358,
    label: 'Kumba, Cameroon'
  },
  {
    value: 359,
    label: 'Freetown, Sierra Leone'
  },
  {
    value: 360,
    label: 'Koidu, Sierra Leone'
  },
  {
    value: 361,
    label: 'Makeni, Sierra Leone'
  },
  {
    value: 362,
    label: 'Ouagadougou, Burkina Faso'
  },
  {
    value: 363,
    label: 'Bobo Diolasso, Burkina Faso'
  },
  {
    value: 364,
    label: 'Cotonou, Benin'
  },
  {
    value: 365,
    label: 'Porto Novo, Benin'
  },
  {
    value: 366,
    label: 'Parakou, Benin'
  },
  {
    value: 367,
    label: 'Djougou, Benin'
  },
  {
    value: 368,
    label: 'Bohicon, Benin'
  },
  {
    value: 369,
    label: 'Kandi, Benin'
  },
  {
    value: 370,
    label: 'Abomey, Benin'
  },
  {
    value: 371,
    label: 'Natitingou, Benin'
  },
  {
    value: 372,
    label: 'Lome, Togo'
  },
  {
    value: 373,
    label: 'Sokode, Togo'
  },
  {
    value: 374,
    label: 'Monrovie, Liberia'
  },
  {
    value: 375,
    label: 'Ajdabivah, Liberia'
  },
  {
    value: 376,
    label: 'Libreville, Gabon'
  },
  {
    value: 377,
    label: 'Port Gentil. Gabon'
  },
  {
    value: 378,
    label: 'Serekunda, Gambia'
  }
]
