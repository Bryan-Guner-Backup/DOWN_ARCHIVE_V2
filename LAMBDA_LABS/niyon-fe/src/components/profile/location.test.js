import React from 'react'
import toJSON from 'enzyme-to-json'
import { location } from './location'

describe('Testing locations', () => {
  it('should render locations array correctly', () => {
    expect(location).toEqual(expect.arrayContaining(location))
  })

  it('should pass a snapshot test correctly', () => {
    expect(toJSON(location)).toMatchSnapshot()
  })
})
