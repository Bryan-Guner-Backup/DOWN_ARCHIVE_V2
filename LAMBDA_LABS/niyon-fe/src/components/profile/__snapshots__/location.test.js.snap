// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing locations should pass a snapshot test correctly 1`] = `undefined`;
